[Docs]Update Kubernetes (Helm) (#11406)
apache/pulsar: site2/docs/helm-overview.md

To get up and run with these charts as fast as possible, in a **non-production** use case, we provide a [quick start guide](getting-started-helm.md) for Proof of Concept (PoC) deployments.

> This guide walks the user through deploying these charts with default values and features, but does not meet production ready requirements. To deploy these charts into production under sustained load, follow the complete [Installation Guide](helm-install.md).

Reviewer: ```suggestion
This guide walks you through deploying these charts with default values and features; the default settings do not meet the production ready requirements. To deploy these charts into production under sustained load, follow the [installation guide](helm-install.md).
```

Reviewer: Try to use second person, for details, see https://developers.google.com/style/person

```

For the purposes of continuity, these charts have some Kubernetes objects that cannot be removed when performing `helm delete`.

> It is recommended to  remove these items, as they affect re-deployment.

Reviewer: ```suggestion
It is recommended to remove these items, as they affect re-deployment.
```
